curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers.english.md

```diff
@@ -30,6 +34,10 @@ tests:
     testString: assert(typeof fiboEvenSum(10) === 'number');
   - text: <code>fiboEvenSum(10)</code> should return 10.
     testString: assert.strictEqual(fiboEvenSum(10), 10);
+  - text: <code>fiboEvenSum(8)</code> should return 10.
```

Comment:
```suggestion
  - text: Your function should sum the even-valued Fibonacci numbers: <code>fiboEvenSum(8)</code> should return 10.
```
Hi, could you use this wording? It's the one we agreed on in https://github.com/freeCodeCamp/freeCodeCamp/issues/38753#issuecomment-634636674

Reply:
@ojeytonwilliams Ok I will use the wording